### ISSUE TITLE
Missing the message error when deploying an image

### ISSUE THREAD

**Issue by ausias-armesto** [cdklabs/cdk-ecr-deployment#48]
Hi, 
I'm trying to use this construct to build an image from my CDK application but I got the following message error with no more information of what is the problem. I suppose that the root cause of the problem is not shown in the console and it is hidden at some point. I'm stacked and do not know where to look for posible causes of the failure.
```
stderr: Bundling asset cicd-infrastructure/Custom::CDKECRDeploymentbd07c930edb94112a20f03f096f53666512MiB/Code/Stage...
Loaded plugins: ovl, priorities
Resolving Dependencies
--> Running transaction check
---> Package btrfs-progs-devel.x86_64 0:4.9.1-2.12.amzn1 will be installed
--> Processing Dependency: btrfs-progs = 4.9.1-2.12.amzn1 for package: btrfs-progs-devel-4.9.1-2.12.amzn1.x86_64
--> Processing Dependency: libbtrfs.so.0()(64bit) for package: btrfs-progs-devel-4.9.1-2.12.amzn1.x86_64
---> Package device-mapper-devel.x86_64 0:1.02.135-1.31.amzn1 will be installed
---> Package gpgme-devel.x86_64 0:1.4.3-5.15.amzn1 will be installed
---> Package libassuan-devel.x86_64 0:2.0.3-3.3.amzn1 will be installed
--> Processing Dependency: pth-devel for package: libassuan-devel-2.0.3-3.3.amzn1.x86_64
---> Package libudev-devel.x86_64 0:173-4.13.amzn1 will be installed
--> Running transaction check
---> Package btrfs-progs.x86_64 0:4.9.1-2.12.amzn1 will be installed
--> Processing Dependency: libext2fs.so.2()(64bit) for package: btrfs-progs-4.9.1-2.12.amzn1.x86_64
---> Package pth-devel.x86_64 0:2.0.7-9.3.7.amzn1 will be installed
--> Running transaction check
---> Package e2fsprogs-libs.x86_64 0:1.43.5-2.44.amzn1 will be installed
--> Processing Dependency: libcom_err(x86-64) = 1.43.5-2.44.amzn1 for package: e2fsprogs-libs-1.43.5-2.44.amzn1.x86_64
--> Running transaction check
---> Package libcom_err.x86_64 0:1.43.5-2.43.amzn1 will be updated
--> Processing Dependency: libcom_err(x86-64) = 1.43.5-2.43.amzn1 for package: libcom_err-devel-1.43.5-2.43.amzn1.x86_64
---> Package libcom_err.x86_64 0:1.43.5-2.44.amzn1 will be an update
--> Running transaction check
---> Package libcom_err-devel.x86_64 0:1.43.5-2.43.amzn1 will be updated
---> Package libcom_err-devel.x86_64 0:1.43.5-2.44.amzn1 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch      Version                 Repository       Size
================================================================================
Installing:
 btrfs-progs-devel      x86_64    4.9.1-2.12.amzn1        amzn-main        54 k
 device-mapper-devel    x86_64    1.02.135-1.31.amzn1     amzn-main       192 k
 gpgme-devel            x86_64    1.4.3-5.15.amzn1        amzn-main       110 k
 libassuan-devel        x86_64    2.0.3-3.3.amzn1         amzn-main        97 k
 libudev-devel          x86_64    173-4.13.amzn1          amzn-main        92 k
Installing for dependencies:
 btrfs-progs            x86_64    4.9.1-2.12.amzn1        amzn-main       1.9 M
 e2fsprogs-libs         x86_64    1.43.5-2.44.amzn1       amzn-updates    208 k
 pth-devel              x86_64    2.0.7-9.3.7.amzn1       amzn-main        55 k
Updating for dependencies:
 libcom_err             x86_64    1.43.5-2.44.amzn1       amzn-updates     47 k
 libcom_err-devel       x86_64    1.43.5-2.44.amzn1       amzn-updates     36 k

Transaction Summary
================================================================================
Install  5 Packages (+3 Dependent packages)
Upgrade             ( 2 Dependent packages)

Total download size: 2.7 M
Downloading packages:
--------------------------------------------------------------------------------
Total                                              480 kB/s | 2.7 MB  00:05
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : libcom_err-1.43.5-2.44.amzn1.x86_64                         1/12
  Installing : e2fsprogs-libs-1.43.5-2.44.amzn1.x86_64                     2/12
  Installing : btrfs-progs-4.9.1-2.12.amzn1.x86_64                         3/12
  Installing : pth-devel-2.0.7-9.3.7.amzn1.x86_64                          4/12
  Installing : libassuan-devel-2.0.3-3.3.amzn1.x86_64                      5/12
  Installing : btrfs-progs-devel-4.9.1-2.12.amzn1.x86_64                   6/12
  Updating   : libcom_err-devel-1.43.5-2.44.amzn1.x86_64                   7/12
  Installing : gpgme-devel-1.4.3-5.15.amzn1.x86_64                         8/12
  Installing : device-mapper-devel-1.02.135-1.31.amzn1.x86_64              9/12
  Installing : libudev-devel-173-4.13.amzn1.x86_64                        10/12
  Cleanup    : libcom_err-devel-1.43.5-2.43.amzn1.x86_64                  11/12
  Cleanup    : libcom_err-1.43.5-2.43.amzn1.x86_64                        12/12
  Verifying  : btrfs-progs-4.9.1-2.12.amzn1.x86_64                         1/12
  Verifying  : libassuan-devel-2.0.3-3.3.amzn1.x86_64                      2/12
  Verifying  : pth-devel-2.0.7-9.3.7.amzn1.x86_64                          3/12
  Verifying  : libudev-devel-173-4.13.amzn1.x86_64                         4/12
  Verifying  : btrfs-progs-devel-4.9.1-2.12.amzn1.x86_64                   5/12
  Verifying  : device-mapper-devel-1.02.135-1.31.amzn1.x86_64              6/12
  Verifying  : libcom_err-devel-1.43.5-2.44.amzn1.x86_64                   7/12
  Verifying  : libcom_err-1.43.5-2.44.amzn1.x86_64                         8/12
  Verifying  : gpgme-devel-1.4.3-5.15.amzn1.x86_64                         9/12
  Verifying  : e2fsprogs-libs-1.43.5-2.44.amzn1.x86_64                    10/12
  Verifying  : libcom_err-devel-1.43.5-2.43.amzn1.x86_64                  11/12
  Verifying  : libcom_err-1.43.5-2.43.amzn1.x86_64                        12/12

Installed:
  btrfs-progs-devel.x86_64 0:4.9.1-2.12.amzn1
  device-mapper-devel.x86_64 0:1.02.135-1.31.amzn1
  gpgme-devel.x86_64 0:1.4.3-5.15.amzn1
  libassuan-devel.x86_64 0:2.0.3-3.3.amzn1
  libudev-devel.x86_64 0:173-4.13.amzn1

Dependency Installed:
  btrfs-progs.x86_64 0:4.9.1-2.12.amzn1
  e2fsprogs-libs.x86_64 0:1.43.5-2.44.amzn1
  pth-devel.x86_64 0:2.0.7-9.3.7.amzn1

Dependency Updated:
  libcom_err.x86_64 0:1.43.5-2.44.amzn1
  libcom_err-devel.x86_64 0:1.43.5-2.44.amzn1

Complete!
CGO_ENABLED=0 CGO_CFLAGS="" CGO_LDFLAGS="-L/usr/lib64 -lgpgme" go build "-buildmode=pie" -gcflags "" -tags "exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp" -o /asset-output/main
go: downloading github.com/aws/aws-lambda-go v1.23.0
......
......
......
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lego@1.9.0 command:setup:cicd: `node --nolazy -r ts-node/register/transpile-only src/index.ts setup-cicd`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lego@1.9.0 command:setup:cicd script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\aarmesto\AppData\Roaming\npm-cache\_logs\2021-05-12T14_16_57_749Z-debug.log

````

My code snipptet is the following: 

````
    const dockerImage  = new DockerImageAsset(this, 'docker-asset', {
      directory: path.join(__dirname, '../../../directoryWithDockerfile/')
    });

    new ecrdeploy.ECRDeployment(this, 'docker-image', {
      src: new ecrdeploy.DockerImageName(dockerImage.imageUri),
      dest: new ecrdeploy.DockerImageName(`${process.env.AWS_ACCOUNT_ID}.dkr.ecr.${process.env.AWS_REGION}.amazonaws.com/devops-events:latest`),
    });
````

I'm using version 0.0.14 and I have used changing @aws-cdk to different versions (1.92.0 , 1.98.0 , 1.101.0) but without success

**Comment by wchaws** [comment]
It looks like the error in "lego@1.9.0 command:setup:cicd: `node --nolazy -r ts-node/register/transpile-only src/index.ts setup-cicd`"

**Comment by ausias-armesto** [comment]
Thanks @wchaws for answering, but that command is the one I used to launch the CDK code. NPM is informing that the launcher program has failed, but does not know the root cause. If I comment the code snippet from above it works fine again, that is why I releated to the cdk-ecr-deployment

**Comment by wchaws** [comment]
This is what I get by running `npx cdk synth -a "npx ts-node -P tsconfig.jest.json --prefer-ts-exts test/integ.ecr-deployment.ts"`. It really doesn't have any additional logs.
```
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/prometheus/common v0.10.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
go: downloading github.com/prometheus/procfs v0.2.0
go: downloading github.com/cespare/xxhash/v2 v2.1.1
***************************************************
*** Newer version of CDK is available [1.103.0] ***
*** Upgrade recommended                         ***
***************************************************
Resources:
  NginxRepo42B51311:
    Type: AWS::ECR::Repository
    Properties:
      RepositoryName: nginx
    UpdateReplacePolicy: Delete
    DeletionPolicy: Delete
    Metadata:
      aws:cdk:path: test-ecr-deployments/NginxRepo/Resource
```
